### DISCUSSION TITLE
✅ Widget Activate/Deactivate option - [ED-12546]

### DISCUSSION THREAD

**Discussion by sarmadali777** [elementor/elementor#15378] [Feature Request]
Please allow users to activate only those widgets which they want to use on their website. Similarly "Essential Addons For Elementor". I think it will improve website speed and overall performance.

**Comment by aaliyahfraser** [comment]
By deactivating widgets that you are not using on your WordPress site, you can potentially increase the speed of your website. When you deactivate a widget, it is removed from the sidebar or other widgetized area of your site, so the code that powers the widget is no longer being loaded on every page of your site. This can reduce the amount of code that needs to be loaded and processed, which can improve the overall performance and speed of your site.

To deactivate a widget, you can go to the Widgets section of your WordPress dashboard, which is usually found under Appearance > Widgets. From here, you can click on the widget that you want to deactivate, and then click on the "Deactivate" button. This will remove the widget from your site, and it will no longer be displayed to your visitors.

Note that deactivating widgets will not have a significant impact on the speed of your site unless you have a large number of inactive widgets. In general, it's a good idea to regularly review your widgets and deactivate any that you are not using, as this can help to keep your site running smoothly and efficiently. If you need more widgets for your Elementor WordPress website you can use Elementor addons. here is list of  **[Best Elementor Addons and Plugins for WordPress 2023.](https://quomodosoft.com/best-elementor-addons-and-plugins/)**

**Comment by 79ho3ein** [comment]
I think this feature does not have a noticeable effect on increasing the speed of the site. **But it will definitely increase the loading speed of the editor.**
Anyway, this is a great and useful feature. I had already discussed this with an Elementor developer. It seems that the reason why Elementor does not add this feature is that they do not have the necessary infrastructure to add this feature.
I hope with the release of V2 of the Elementor editor, it will be possible to add this feature.

**Comment by nicholaszein** [comment]
Hi,

this feature was introduced in Elementor version 3.18.0!

🥳
